demo/site/src/util/graphQLClientMiddleware.ts

```diff
@@ -0,0 +1,15 @@
+import { createFetchWithDefaults, createGraphQLFetch } from "@comet/cms-site";
+
+export function createGraphQLFetchMiddleware() {
+    return createGraphQLFetch(
+        createFetchWithDefaults(fetch, {
+            next: {
+                revalidate: 7.5 * 60,
+            },
+            headers: {
+                authorization: `Basic ${Buffer.from(`vivid:${process.env.API_PASSWORD}`).toString("base64")}`,
+            },
+        }),
+        `${process.env.API_URL_INTERNAL}/graphql`,
+    );
+}
```

Comment:
Might createGraphQLFetch from demo/site/src/util/graphQLClient.ts somehow be reused?